src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    list-style-type: none;
+    display: flex;
+    padding-inline-start: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 60px;
+    color: #060b35;
+    text-decoration: none;
+    font-style: normal;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+
+  &__link:hover {
+    color: $color-content;
```

Comment:
Ensure that the variable `$color-content` is defined in your SCSS files. If it's not defined, this will cause a compilation error.